WebShop/WebShop.Frontend/Frontend/src/pages/ItemPage.js

```diff
@@ -1,99 +1,85 @@
-import React, { useState } from 'react'
-import { useParams } from 'react-router'
-// import MainBar from '../components/MainBar';
+import axios from 'axios';
+import React, { Component } from 'react'
 
-const ItemPage = () => {
-
-    const [quantity, setQuantity] = useState(1); // '' is the initial state value
-
-    const item = {
-        id: 3981239812,
-        name: 'Sava Sumanovic - Šid',
-        placeHolderImg: "https://mediasfera.rs/wp-content/uploads/2019/01/sava-2-1200x894.jpg",
-        cost: 15,
-        amount: 80,
-        description: "This is the best item you can find on the market, eventualy you can find it on buvljak on Klisa, Novi Sad :).",
+class ItemPage extends Component {
```

Comment:
Ne znam sta je sad u modi u Reactu, al cini mi se da su se forsirale funkcijske komponente umesto klasne. Ne 
treba menjati sad realno, al ono cisto sta je jace za ubuduce da koristimo.

Reply:
u vezi sa ovim je velika dilema sta koristiti, probao sam bio i jedno i drugo. Kod funkcionalnih komponenti se malo drugacije koriste hook-ovi pomocu useEffect-a i nisam se najbolje bio snasao sa njima pa sam ovako odradio samo zato jer sam zurio da to odradim jer sam bio na poslu. Sigurno da moze i sa funkcionalnim komponentima.

Reply:
Sve prezentacione komponente koje nemaju nikakvu logiku osim vizuelnog prikaza treba da nam budu funkcijske, tj. one koje nemaju state.